sdmetrics/statistical/utils.py

```diff
@@ -2,27 +2,27 @@
 from collections import Counter
 
 
-def frequencies(real, fake):
+def frequencies(real, synthetic):
     """
     Given two iterators containing categorical data, this transforms it into
     observed/expected frequencies which can be used for statistical tests. It
-    adds a regularization term to handle cases where the fake data contains
+    adds a regularization term to handle cases where the synthetic data contains
```

Comment:
is "regularization" the right word here?

Reply:
Additive smoothing / Laplace smoothing / pseudo-counts is often treated as regularization so I think it's fine here.